admin/metabox/class-metabox.php

```diff
@@ -443,6 +443,12 @@ function do_meta_box( $meta_field_def, $key = '' ) {
 			$placeholder = $meta_field_def['placeholder'];
 		}
 
+		$aria_describedby = $description = '';
+		if ( isset( $meta_field_def['description'] ) ) {
+			$aria_describedby = ' aria-describedby="' . esc_attr( $esc_form_key ) . '-desc"';
```

Comment:
escaping the esc_form_key already happens at line 433. Don't think it needs to be escaped again unless I'm missing something.